### PR TITLE
License updates: release as LGPL! (closes #16)

### LINKED ISSUE #16: License update for open sourcing
Things to do or check:
- [x] make sure LGPL v2.1 is correct (instead of v2)
- [x] in sketch_tool/package.json:
  - [x] private: true => (remove)
  - [x] license: "LGPL-2.1" (the SPDX identifier for LGPL 2.1)
  - [x] copyright: add if not there? [omitting; FB doesn't use one in theirs]
  - [x] make sure jspm deps are included/handled correctly
- [x] sketch_tool LICENSE file (or remove and just use the main repo one)
- [x] main repository LICENSE file
- [x] make sure Python code has proper licensing too
- [x] check references in the docs?
- [x] make sure copyright statement follows MIT's format everywhere
- [x] update license info in sketch tool "info& legal" window

### PR DESCRIPTION
**Pull request by msegado**
See #16 for the current TODO list and status. I'll merge into master once everything is ready.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,2 +1,996 @@
...
+
+This library is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License along
+with this library (copied below); if not, write to the Free Software Foundation,
+Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA.
+
+Other licensing options may be available if required; contact the MIT Technology
```
Comment (msegado) [inline LICENSE]: @jefrench, is it ok by you to make this explicit like I do here? I figure that gives people a path forward if LGPL doesn't work for them for some reason. (I suspect TLO will contact you if someone does ever get in touch with them, btw.)

Comment (msegado) [inline LICENSE]: (i.e., the "Other licensing options may be available if required" blurb)

Comment (jefrench) [inline LICENSE]: I don't think that we can do this. I'm going to want to run this by TLO before we make such an open ended statement. I think we have permission to release it under LGPL 2.1, but no other license. I'm not very clear on the legality of these things. I might make this statement a bit weaker. Also, I don't think TLO necessarily wants us to list them as a contact for this.

Comment (msegado) [inline LICENSE]: Yes, that's a good point - I'll just remove that line for now and we can always re-add it later if we want it in there!

**Pull request merged**